packages/fiori-elements-writer/test/__snapshots__/feop.test.ts.snap

```diff
@@ -3689,30 +3702,32 @@ archive.zip
   },
   "tsconfig.json": Object {
     "contents": "{
-  \\"compilerOptions\\": {
-    \\"target\\": \\"es2015\\",
-    \\"module\\": \\"es2015\\",
-    \\"skipLibCheck\\": true,
-    \\"preserveConstEnums\\": true,
-    \\"inlineSourceMap\\": true,
-    \\"allowJs\\": true,
-    \\"strict\\": true,
-    \\"strictNullChecks\\": true,
-    \\"strictPropertyInitialization\\": false,
-    \\"moduleResolution\\": \\"node\\",
-    \\"rootDir\\": \\"webapp\\",
-    \\"outDir\\": \\"./dist\\",
-    \\"baseUrl\\": \\"./\\",
-    \\"paths\\": {
-      \\"fefeop2ts/*\\": [
-        \\"webapp/*\\"
-      ]
+    \\"compilerOptions\\": {
```

Comment:
We also polished the `tsconfig.json` in our latest example:
https://github.com/SAP-samples/ui5-typescript-helloworld/blob/main/tsconfig.json

* `es2022` supported by Tooling V3, 
* some config is useless when Babel is used, e.g. `inlineSourceMaps`, `outDir`
* if `strict` is `true` all strict rules are true and thus `strictNullChecks` is redundant defined here
* if `types` are used, you may also use `typeRoots` to point to the definitely typed types like in the hello world.
* the `composite` flag in the helloworld sample is not needed (still a rest from the cleanup)

Reply:
@petermuessig https://github.com/SAP/open-ux-tools/pull/1036/commits/065699ecee2bafc20d6113da46b9edcdfa47f88a takes your suggestions into account